### ISSUE TITLE
RC5 ion-item-sliding doesn't snap back to correct position when pulled left or right.

### ISSUE THREAD

**Issue by mtycholaz** [ionic-team/ionic#10048]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ x ] **2.x**

**I'm submitting a ...**  (check one with "x")
[x ] bug report
[ ] feature request

**Current behavior:**
Upgraded from RC4 to RC5. When sliding an item left or right in the list, the item will track with the position of my finger, instead of bouncing back to the correct position. When I lift my finger, the ion-item stays where I slide it to. Setup looks like this

        <ion-list [virtualScroll]="filteredMessages" [virtualTrackBy]="trackBy" approxItemHeight="86px" no-padding no-margin>
        <ion-item-sliding *virtualItem="let message">            
            <ion-item class="item-message" >
                <ion-avatar item-left>
                    
                </ion-avatar>
                <h2>
                    contact name
                </h2>
                <p>note</p>
                <small>duration</small>
                <ion-note item-right>
                    <small>test message</small>
                    <br/>
                    <i class="fa fa-2" aria-hidden="true"></i>
                </ion-note>
            </ion-item>
            <ion-item-options side="left">
                <button ion-button color="danger">
                    <i class="fa fa-trash fa-3"></i>
                    Archive
                </button>
            </ion-item-options>

            <ion-item-options side="right">
                <button ion-button color="secondary">
                    <i class="fa fa-bookmark fa-3"></i>
                    Save
                </button>
                <button ion-button>
                    <i class="fa fa-flag fa-3"></i>
                    Read
                </button>
                <button ion-button>
                    <i class="fa fa-exclamation-circle fa-3"></i>
                    Urgent
                </button>
            </ion-item-options>

        </ion-item-sliding>
    </ion-list>

**Expected behavior:**
The ion-item should bounce/snap back to the correct position if pulled beyond the limit.

**Steps to reproduce:**
Tried to make a plunker, but RC5 doesn't work in Edge :( 404 errors for .ts files

**Other information:**
Windows 10 mobile

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Your system information:

 You have been opted out of telemetry. To change this, run: cordova telemetry on.
6.4.0

Ionic Framework Version: 2.0.0-rc.5
Ionic CLI Version: 2.1.12
Ionic App Lib Version: 2.1.7
Ionic App Scripts Version: 0.0.47
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v6.9.1
Xcode version: Not installed


******************************************************
 Dependency warning - for the CLI to run correctly,
 it is highly recommended to install/upgrade the following:

 Please install your Cordova CLI to version  >=4.2.0 `npm install -g cordova`

******************************************************
```

**Comment by gigocabrera** [comment]
+1 
I'm having a similar issue

**Comment by gigocabrera** [comment]
Rolled back to RC4 and the issue goes away

**Comment by masimplo** [comment]
This seems to have been fixed in 2.0.0 final.
